docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -132,39 +136,39 @@ Here you can specify port ranges that should be filtered.
 
 #### Log removed HTML events
 
-If enabled, AdGuard records blocked HTML elements in the filtering log.
+If enabled, AdGuard records blocked HTML elements in *Recent activity*.
 
 #### Scriplet debugging
 
-If you need to activate debugging of scriptlets, enable this feature. Then there will be messages in the browser log that some scriplet rules have been applied.
+This options displays information about applied scriptlet rules to the browser log.
 
 #### Excluded apps
 
-Here you can list package names and UIDs that you want to exclude from AdGuard protection.
+Here you can list package names and UIDs that you want to exclude from AdGuard's protection.
 
 #### QUIC bypass packages
 
 Here you can specify package names for which AdGuard should bypass QUIC traffic.
 
 #### Reconfigure Automatic proxy when network changes
 
-Enable this setting if you want the protection to restart to reconfigure the automatic proxy settings when the device connects to another network. The state of this setting affects operation only if the current routing mode is Automatic proxy.
+This option restarts protection to reconfigure the automatic proxy settings when the device connects to another network. It is only applied if *Automatic proxy* is selected in *Routing mode*.
 
 #### IPv6 filtering
 
-If enabled, AdGuard filters IPv6 networks if an IPv6 network interface is available.
+This option makes AdGuard filter IPv6 networks if an IPv6 network interface is available.
 
 #### IPv4 ranges excluded from filtering
 
-Filtering for IPv4 ranges, listed in this section, will be disabled.
+Filtering for IPv4 ranges listed in this section is disabled.
 
 #### IPv6 ranges excluded from filtering
 
-Filtering for IPv6 ranges, listed in this section, will be disabled.
+Filtering for IPv6 ranges listed in this section is disabled.
 
 #### TCP keepalive for outgoing sockets
 
-If enabled, AdGuard sends a keepalive probe after the specified time period to ensure that the TCP connection is alive. Here you can specify the TCP keepalive idle time before starting keepalive probes and time between keepalive probes for an unresponsive peer.
+This option makes AdGuard send a keepalive probe after the specified time period to ensure that the TCP connection is alive. Here you can specify the idle time before starting keepalive probes and time between keepalive probes for an unresponsive peer.
```

Comment:
If this is enabled, AdGuard will send a keepalive probe after the specified time period to ensure that the TCP connection is alive. Here, you can specify the idle time before starting keepalive probes and the time between keepalive probes for an unresponsive peer.